### PR TITLE
Add Hub namespace for statisticswiki T3534

### PR DESCRIPTION
**Pull request by Amanda-Catherine**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -2896,6 +2896,10 @@
 			NS_BOILERPLATE => 'Boilerplate',
 			NS_BOILERPLATE_TALK => 'Boilerplate_talk',			
 		),
+		'statisticswiki' => array(
```
Comment (paladox) [inline LocalSettings.php]: need to define the namespace.

Comment (Amanda-Catherine) [reply LocalSettings.php]: @paladox Huh?

Comment (paladox) [reply LocalSettings.php]: See where we use define() at top of LS

Comment (Amanda-Catherine) [reply LocalSettings.php]: Done

**Pull request merged**